config.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+options:
+  container:
+    type: string
+    description: The name of the Azure Storage Container
+  storage-account:
+    type: string
+    description: The name of the Azure storage account
+  path:
+    type: string
+    description: The path inside the bucket/container to store objects.
+    default: ''
+  connection-protocol:
+    type: string  
+    description: The storage protocol to use when connecting to Azure storage
+    default: 'abfss'
+  credentials:
+    type: secret
```

Comment:
`type: secret` so that Juju automatically validates strings that are not valid secrets.